src/main/java/org/carlspring/cloud/storage/s3fs/S3AccessControlList.java

```diff
@@ -87,5 +81,4 @@ private String fileName()
     {
         return fileStoreName + S3Path.PATH_SEPARATOR + key;
     }
-
-}
+}
```

Comment:
Cosmetic: No new line at the end of the file.

Reply:
Done.